integration-tests/src/test/resources/features/processor.feature

```diff
@@ -30,3 +31,92 @@ Feature: Processor tests
     And the Manager metric 'manager_processor_status_change_total{status="PROVISIONING",}' count is at least 1
     And the Manager metric 'manager_processor_status_change_total{status="READY",}' count is at least 1
     And the Manager metric 'manager_processor_status_change_total{status="DELETED",}' count is at least 1
+
+  Scenario: Processor filter is updated
```

Comment:
Please move the scenario to `processor-filter.feature` to keep such tests separate from "plain" processor tests.